src/Stratis.Bitcoin.IntegrationTests/BlockStore/RetrieveFromBlockStoreSteps.cs

```diff
@@ -56,16 +52,13 @@ protected override void AfterTest()
 
         private void a_pow_node_running()
         {
-            this.node = this.builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-            this.node.Start();
+            this.node = this.builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
         }
 
         private void a_pow_node_to_transact_with()
         {
-            this.transactionNode = this.builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-            this.transactionNode.Start();
-
-            this.transactionNode.CreateRPCClient().AddNode(this.node.Endpoint, true);
+            this.transactionNode = this.builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
+            TestHelper.Connect(this.transactionNode, this.node);
```

Comment:
Why not ConnectAndSync

Reply:
At this point the first node has not yet mined any blocks so ConnectAndSync has no benefit here 👍